vendor/k8s.io/kubernetes/pkg/kubectl/cmd/expose.go

```diff
@@ -273,7 +273,7 @@ func RunExpose(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command, args []str
 			return f.PrintObject(cmd, mapper, object, out)
 		}
 
-		cmdutil.PrintSuccess(mapper, false, out, info.Mapping.Resource, info.Name, "exposed")
+		cmdutil.PrintSuccess(mapper, false, out, info.Mapping.Resource, info.Name, cmdutil.GetDryRunFlag(cmd), "exposed")
```

Comment:
we're already returning above:
```
if cmdutil.GetDryRunFlag(cmd) {
  return f.PrintObject(cmd, mapper, object, out)
}
```

should we be doing this above?
```
if cmdutil.GetDryRunFlag(cmd) {
  if len(cmdutil.GetFlagString(cmd, "output")) > 0 {
     return f.PrintObject(cmd, mapper, object, out)
  }
  cmdutil.PrintSuccess(mapper, false, out, info.Mapping.Resource, info.Name, true, "exposed")
  return nil
}
```

and just passing `false` here?